lib/modules/versioning/distro.ts

```diff
@@ -58,7 +100,58 @@ export class DistroInfo {
     return input;
   }
 
+  /**
+   * Get schedule of a given release
+   * @param input A codename/semVer
+   * @returns A schedule if available, otherwise undefined
+   */
   public getSchedule(input: string): DistroSchedule {
-    return this._distroInfo[input];
+    const ver = this.getVersionByCodename(input);
+    return this._distroInfo[ver];
```

Comment:
```suggestion
    return this._distroInfo[ver] ?? null;
```

We prefer `null` over `undefined`